.github/workflows/swift.yaml

```diff
@@ -0,0 +1,41 @@
+name: Swift - Workflow - Cocoapods (Xcode 11.2.1)
+
+on: [push]
+
+jobs:
+  bundle:
```

Comment:
I don't think this job is going to do what you expect. Each job runs in its own VM instance. I think you need to set up a cache to get the output. See these docs:
- https://github.com/actions/cache/blob/master/examples.md#swift-objective-c---cocoapods
- https://help.github.com/en/actions/configuring-and-managing-workflows/caching-dependencies-to-speed-up-workflows

Reply:
Yes! Looks like we should use `artifacts` to carry data from one job to another.

Reply:
I was hoping for something like Google Cloud Builder's `workspace` which carries a dir from one job to the next.

Reply:
That's what the cache action does.